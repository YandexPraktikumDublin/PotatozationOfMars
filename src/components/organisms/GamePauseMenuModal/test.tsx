import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { GamePauseMenuModal } from '.'

describe('<GamePauseMenuModal />', () => {
  const toggleModal = jest.fn()
  it('should renders correct <GamePauseMenuModal />', () => {
    const wrapper = shallow(<GamePauseMenuModal toggleModal={toggleModal} />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
