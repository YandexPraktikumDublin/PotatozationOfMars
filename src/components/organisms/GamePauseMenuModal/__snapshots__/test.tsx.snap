// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GamePauseMenuModal /> should renders correct <GamePauseMenuModal /> 1`] = `
<Modal
  toggleModal={[MockFunction]}
>
  <Title>
    Pause
  </Title>
  <GamePauseMenu
    toggleModal={[MockFunction]}
  />
</Modal>
`;
